tests/test_bs.py

```diff
@@ -147,6 +147,23 @@ def setup_bootstrap_in_namespace(namespace, bs_deployment_info, bootstrap_config
 
     bs_pod['key'] = match.group('bootstrap_key')
     bs_deployment_info.pods = [bs_pod]
+
+    match = pod.search_phrase_in_pod_log(bs_pod['name'], namespace, 'poet',
+                                         "REST proxy start listening on 0.0.0.0:80")
+    if not match:
+        raise Exception("Failed to read container logs in {0}".format("poet"))
+
+    match = pod.search_phrase_in_pod_log(bs_pod['name'], namespace, 'bootstrap',
+                                             "App started.")
+    if not match:
+        raise Exception("Failed to read container logs in {0}".format("bootstrap"))
+
+
+    print("Starting PoET")
+    out = api_call(bs_pod['pod_ip'], '{ "nodeAddress": "127.0.0.1:9091" }',  'v1/start', namespace, "80")
```

Comment:
port 80 should be a number (not string)

Reply:
eventually, it is used as a string in an IP:PORT string pair. I don't see a point casting back and forth here.